### PR TITLE
Added Revenue Data Generator

### PR DESCRIPTION
**Pull request by yakkomajuri**
## Changes

*Please describe.*  
*If this affects the front-end, include screenshots.*  

Built this quickly to generate some data for a tutorial. Can definitely be iterated on, but this was just a quick thing I built for myself and thought it might be useful to share.

Can be ran like:

**Postgres**
`DEBUG=1 python manage.py generate_revenue_data --org="Test" --team_name="HogFlix Demo App"`

**CH**
`DEBUG=1 PRIMARY_DB=clickhouse DATABASE_URL=postgres://posthog:posthog@localhost:5439/posthog  python manage.py generate_revenue_data --org="Test" --team_name="HogFlix Demo App" --use_ch=True`

It was the case for me at first that the DB had multiple orgs and multiple teams with the same name. Can be the cause of confusion when events don't appear after running the command.

I'll also probably improve our demo data generation soon as well.

Gets you some nice (yet very simple) data to work with:

Screenshot 2020-11-12 at 17 25 30
Screenshot 2020-11-12 at 17 25 56
Screenshot 2020-11-12 at 17 26 18

### Potential further improvements

- Also give each person some pageviews and things to track: 
    - Retention by plan
    - Conversion by plan (funnels)

## Checklist

- [ ] All querysets/queries filter by Organization, Team, and User (if this PR affects ANY querysets/queries).
- [ ] Django backend tests (if this PR affects the backend).
- [ ] Cypress end-to-end tests (if this PR affects the frontend).

### CODE REVIEW

File: posthog/management/commands/generate_revenue_data.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+                properties={"plan": plan[0], "purchase_value": plan[1],},
+                timestamp=now() - relativedelta(days=random.randint(0, 100)),
+                event_uuid=event_uuid,
+            )
+
+    def _get_team(self, org, team_name, team_id):
+        organization = Organization.objects.filter(name=org)[0]
+        try:
+            if team_name:
+                team = organization.teams.filter(name=team_name[0])[0]
```
Comment (EDsCODE) [inline posthog/management/commands/generate_revenue_data.py:96]: can unwrap team_name[0] higher up like you did with org so the syntax here is cleaner

**Review by EDsCODE — APPROVED** [review]
lgtm

**Comment by mariusandra** [comment]
Drive by comment below.

This would be a cool starting point to teach people about the [`posthog-currency-normalization-plugin`](https://github.com/PostHog/posthog-currency-normalization-plugin).

**Comment by yakkomajuri** [comment]
Made the change @EDsCODE and also made it more configurable in terms of date range and number of events to create.

@mariusandra yeah that's a great point - will mention the plugin in the tutorial.

**Pull request merged**